openmetadata-ui/src/main/resources/ui/src/constants/constants.ts

```diff
@@ -33,7 +33,7 @@ export const INITIAL_PAGING_VALUE = 1;
 export const JSON_TAB_SIZE = 2;
 export const PAGE_SIZE = 10;
 export const PAGE_SIZE_BASE = 12;
-export const PAGE_SIZE_MEDIUM = 16;
```

Comment:
what will happen if we are using count 16 somewhere?  did you check that scenario?

Reply:
It was used in 2 places where we were also intending to change the size.